scrapy/downloadermiddlewares/httpcompression.py

```diff
@@ -10,10 +10,13 @@
 ACCEPTED_ENCODINGS = [b'gzip', b'deflate']
```

Comment:
You can change this to a `set` as we're performing many `in` (pertinence checks) and this operation is faster on `set` than on a `list`